### PR TITLE
patches: update calendar for 1.17/1.18/1.19 patches releases

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

Update patches release schedule for 1.17/1.18/1.19

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

/assign @saschagrunert @hasheddan @justaugustus 
cc @kubernetes/release-managers

### CODE REVIEW

File: releases/patch-releases.md [hunk 1]
```diff
@@ -57,26 +57,29 @@ releases may also occur in between these.
...
 ### 1.19
 
-Next patch release is **1.19.4**
+Next patch release is **1.19.5**
 
 End of Life for **1.19** is **2021-09-30**
 
 | PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
 |--- |--- |--- |
+| 1.19.5        | 2020-12-11           | 2020-12-1   |
```
Comment (saschagrunert) [inline releases/patch-releases.md:66]: We also cut 1.20.0-rc.0 at that date. I would prefer moving it to another date. How about December 9th?

File: releases/patch-releases.md [hunk 2]
```diff
@@ -57,26 +57,29 @@ releases may also occur in between these.
...
 
-Next patch release is **1.18.11**
+Next patch release is **1.18.13**
 
 End of Life for **1.18** is **2021-04-30**
 
 | PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
 |--- |--- |--- |
-| 1.18.11       | 2020-11-06           | 2020-11-11  |
+| 1.18.13       | 2020-12-11           | 2020-12-16  |
```
Comment (saschagrunert) [inline releases/patch-releases.md:80]: The date does not seem to be aligned with  v1.19.5, what is the reason for that?

Comment (cpanato) [reply releases/patch-releases.md:80]: ops...

File: releases/patch-releases.md (hunk 1 above)
Comment (cpanato) [reply releases/patch-releases.md:66]: sure!

File: releases/patch-releases.md (hunk 2 above)
Comment (cpanato) [reply releases/patch-releases.md]: done, thanks!

PTAL

**Review by saschagrunert — APPROVED** [review]
/lgtm

/hold
Please take a look @justaugustus

**Review by hasheddan — APPROVED** [review]
/lgtm

**Review by justaugustus — CHANGES_REQUESTED** [review]
@cpanato -- We'll want to also drop the November from the "Upcoming Monthly Releases" section

**Comment by justaugustus** [comment]
Thanks!
/lgtm
/approve
/hold cancel

**Pull request merged**